its_the_way.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+
+def start_with_deep(path: str):
+    return [file for file in os.listdir(path) if file.startswith("deep")]
+
+
+start_with_deep("C:\\Users\\dvirp\\OneDrive - edu.hac.ac.il\\Documents\\GitHub\\Notebooks-master\\week05\\images")
```

Comment:
Make sure this code doesnt execute when importing this file - read this and change 
https://www.freecodecamp.org/news/if-name-main-python-example/